lms/static/styles/components/_URLPicker.scss

```diff
@@ -1,7 +1,6 @@
 .URLPicker__dialog {
-  display: flex;
-  flex-direction: column;
-  width: 80vw;
+  /* Prevent the Dialog from jumping size when changing state from fetching to fetched. */
```

Comment:
`URLPicker` is the "Enter URL" dialog (see `URLPicker.js`), it doesn't have any fetching/fetched state associated with it.